src/frontend/sharedComponents/ActionRow.tsx

```diff
@@ -113,18 +83,6 @@ export const ActionsRow = ({fieldRefs}: ActionButtonsProps) => {
   return (
     <>
```

Comment:
can remove the fragment here now

Reply:
this component is now just forwarding another component which is not ideal. I think im just going to get rid of it instead